src/utils/markerTypes.js

```diff
@@ -40,6 +41,8 @@ const getNewPrimitive = marker => {
       return new TriangleList();
     case MARKER_OBJECT_TYPES.CUBE_LIST:
       return new CubeList();
+    case MARKER_OBJECT_TYPES.TEXT_VIEW_FACING:
+      return new ViewFacingText(options.text);
```

Comment:
This could directly be from `marker.text`. Options need not be sent to the function. That's a duplication